### ISSUE TITLE
Maintenance Drones Unable to Unweld Disposals Chutes, Bins, and Outlets

### ISSUE THREAD

**Issue by Breenland** [ParadiseSS13/Paradise#9529]
**Problem Description**:
Maintenance drones are unable to unweld any of the disposals devices. Disposals pipes can be removed normally.

**What did you expect to happen**: 
The device to be unwelded after a moment.

**What happened instead**:
The disposals device does not get unwelded, and the welder uses a unit of fuel regardless.

**Why is this bad/What are the consequences:**
Drones cannot move or replace disposal devices, which makes it difficult for them to do that portion of their job. It additionally makes it difficult for drones to complete any projects they may wish to attempt.

**Steps to reproduce the problem**:
1. Be drone
2. Find disposals device (bin, chute, outlet)
3. Attempt to unweld
4. Notice the device is not unwelded, no effect, and a unit of fuel was wasted.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Nothing else I can think of.

**Comment by alex-gh** [comment]
Tried it out, works for me.
Are you sure you did not forget to remove the floor tile underneath?

**Comment by Breenland** [comment]
Yes, I tested it just now on the current live round, and it didn't work. Removed the floor tile, and couldn't wrench or weld the disposal bin, but it still consumed welding fuel. I was able to weld and wrench the trunk beneath it. This applies to devices (bins, chutes, outlets, not pipes) once they've been welded in place only, it's perfectly capable of placing and welding them, just not removing them.
